tests/common/watch_test.go

```diff
@@ -0,0 +1,116 @@
+package common
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.etcd.io/etcd/tests/v3/framework/config"
+	"go.etcd.io/etcd/tests/v3/framework/testutils"
+)
+
+func TestTxn(t *testing.T) {
```

Comment:
The effort is about migrating the test, replacing old test with new one. Which test are you migrating here?
Can you just remove the original code of the test so it will be easier to review and compare.

Reply:
When I was migrating of this part of the test, I found that the watch test actually has a big difference between `e2e` and `integration`.

For example `--interactive` is only for the command line (if I missed something, please let me know, thank you ), but many e2e tests are related to it.

Would like to know your opinion on this issue. Thanks!